mmv1/third_party/terraform/go.mod.erb

```diff
@@ -1,10 +1,9 @@
-<% autogen_exception -%>
```

Comment:
This tag is needed to prevent the ruby compiler from treating this as a normal template.

Reply:
Thanks. Just fixed it.